src/app/events/[slug]/utils/generateStructuredData.ts

```diff
@@ -46,7 +46,7 @@ function getLocation({
 }
 
 export function generateStructuredData(data: Event): WithContext<SchemaEvent> {
-  const {
+export function generateStructuredData(data: Event): WithContext<EventSchema> {
```

Comment:
@CharlyMartin - Make sure when using suggestions to not add/remove unwanted things so that when we commit the suggestion the code doesn't break. 🙏🏼

Reply:
Good catch, thanks!